homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -58,8 +59,6 @@ async def async_setup_entry(
         True,
     )
 
-    return True
-
```

Comment:
This is the only code change: `async_setup_entry` shouldn't be returning a value.